src/Movies.js

```diff
@@ -0,0 +1,37 @@
+import $ from 'jquery';
+
+function parseSingle($row) {
+  let title = $($row).find('td.title > a').text().trim();
+  let titleUrl = $($row).find('td.title > a').attr('href');
+  let rating = parseFloat($($row).find('span.rating-rating').text().trim().split('/')[0]) || undefined;
+  let desc = $($row).find('span.outline').text().trim();
+  let imgUrl = $($row).find('.image img').attr('src');
+  let genre = $($row).find('.title .genre').text().trim();
+
+  return { title, titleUrl, rating, desc, imgUrl, genre };
+}
+
+function renderSingle($output, movie) {
```

Comment:
Yes, that was the initial name, don't know where I was going with this.... probably some inputs on the side ... 😅